subprojects/core/src/main/java/org/gradle/execution/plan/ExecutionPlan.java

```diff
@@ -43,15 +42,22 @@
     void cancelExecution();
 
     /**
-     * <p>Returns the dependencies of a task which are part of the execution plan.</p>
+     * Returns the dependencies of a task which are part of the execution plan.
      *
-     * @return The tasks. Returns an empty set if there are no dependent tasks.
      * @throws IllegalStateException When the task is not part of the execution plan.
      *
-     * @since 4.5
+     * @since 5.1
```

Comment:
Afaik this is not public API, so why are we doing `@since` here?

Reply:
Good question. @lptr Is there a reason or shall we remove `@since`?

Reply:
Let's get rid of `@since` and `@Incubating` in this class since it is not public. For some reason I considered the class public back then when I added the method.